test/src/063-uidmap/main

```diff
@@ -4,15 +4,15 @@ cvmfs_test_name="Uid/Gid Mapping of files and directories"
 cvmfs_run_test() {
   logfile=$1
 
-  local uid_map=$(mktemp)
-  local gid_map=$(mktemp)
+  local uid_map=$(pwd)/uid.map
+  local gid_map=$(pwd)/gid.map
 
   echo "* 1337" > "$uid_map"
   echo "* 1338" > "$gid_map"
   chmod 0444 "$uid_map" "$gid_map"
 
   cvmfs_mount grid.cern.ch "CVMFS_CLAIM_OWNERSHIP=no" \
-    "CVMFS_UID_MAP=$uid_map" "CVMFS_GID_MAP=$gid_map"
+    "CVMFS_UID_MAP=$uid_map" "CVMFS_GID_MAP=$gid_map" "CVMFS_DEBUGLOG=/tmp/cvmfs-test.log"
```

Comment:
`CVMFS_DEBUGLOG` was for debugging only and doesn't need to be in the vanilla test.